### PR TITLE
Tacticool sensors

### PR DESCRIPTION
**Pull request by ZeroNetAlpha**
[Changelogs]: Tacticool Turtlenecks now have sensors for those memes that want to wear them on station as crew.

:cl: ZeroNetAlpha
tweak: Tacticool Turtlenecks are now compliant with Nanotrasen regulations and are equipped with sensors.
/:cl:

[why]: It was a suggestion I saw that I could easily do. Soooo... yeet.

### CODE REVIEW

File: modular_citadel/code/modules/clothing/under/syndicate.dm [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/obj/item/clothing/under/syndicate/tacticool
+	name = "tacticool turtleneck"
+	desc = "Just looking at it makes you want to buy an SKS, go into the woods, and -operate-."
```
Comment (deathride58) [inline modular_citadel/code/modules/clothing/under/syndicate.dm]: You don't need to redefine all of this, only the has_sensor

**Review by deathride58 — CHANGES_REQUESTED** [review]

File: modular_citadel/code/modules/clothing/under/syndicate.dm (hunk 1 above)
Comment (ZeroNetAlpha) [reply modular_citadel/code/modules/clothing/under/syndicate.dm]: Okiedoke

**Comment by ZeroNetAlpha** [comment]
Fixed

**Review by deathride58 — APPROVED** [review]

**Comment by silicons** [comment]
it's supposed to be a tradeoff, smh.

**Comment by cebutris** [comment]
A tradeoff for looking so fabulous

**Pull request merged**